net/tayga/src/opnsense/mvc/app/models/OPNsense/Tayga/General.xml

```diff
@@ -1,27 +1,31 @@
 <model>
     <mount>//OPNsense/tayga/general</mount>
     <description>Tayga configuration</description>
-    <version>0.0.2</version>
+    <version>0.0.3</version>
     <items>
         <enabled type="BooleanField">
             <default>0</default>
             <Required>Y</Required>
         </enabled>
         <v4address type="NetworkField">
             <default>192.168.255.1</default>
-            <Required>N</Required>
+            <Required>Y</Required>
         </v4address>
+        <v4destination type="NetworkField">
+            <default>192.168.254.1</default>
+            <Required>Y</Required>
+        </v4destination>
         <v6address type="NetworkField">
             <default>2001:db8:1::2</default>
-            <Required>N</Required>
+            <Required>Y</Required>
```

Comment:
This should not be "required" and the default should be an empty field. We just need to make sure that the `ipv6-addr` option does not get written to tayga.conf if the field is empty. Tayga will then _generate its IPv6 address by mapping the address specified in ipv4-addr into the NAT64 prefix_ (quote from the man page).

Reply:
I decided to make all required since they will be used in the rc script, there's no smarter way I'm aware of

Reply:
I can't find `v6address` anywhere in the rc script. This is only used for the optional parameter `ipv6-addr` in tayga.conf.